training_config/rc/bidaf.jsonnet

```diff
@@ -20,8 +20,11 @@
       }
     }
   },
-  "train_data_path": "https://allennlp.s3.amazonaws.com/datasets/squad/squad-train-v1.1.json",
-  "validation_data_path": "https://allennlp.s3.amazonaws.com/datasets/squad/squad-dev-v1.1.json",
+  "train_data_path": std.extVar("SQUAD_TRAIN"),
+  "validation_data_path": std.extVar("SQUAD_DEV"),
+  // You can replace the above two lines with these to get the actual squad datasets.
```

Comment:
Adding this same comment in other config files would be nice.  Hmm, looks like you have it in most, but not BART or SWAG.